artifactory/commands/token/revoketoken.go

```diff
@@ -0,0 +1,41 @@
+package token
+
+import (
+	"github.com/jfrog/jfrog-cli-go/artifactory/utils"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+)
+
+type RevokeTokenCommand struct {
+	TokenCommand
+	params    services.RevokeTokenParams
```

Comment:
This comment applies to most of the new token commands:
We try not to use params of types such as *services.RevokeTokenParams* as part of the commands params. This is because *services.RevokeTokenParams* is defined by the jfrog-client-go library, and we don't want to changes the jfrog-client-go API to affect the commands options. See how it is does in other commands as a reference.